src/main/java/org/zalando/fahrschein/NakadiClient.java

```diff
@@ -75,11 +78,15 @@ public Subscription subscribe(String applicationName, String eventName, String c
     }
 
     public <T> void listen(Subscription subscription, Class<T> eventType, Listener<T> listener, StreamParameters streamParameters) throws IOException {
+        listen(subscription, eventType, listener, streamParameters, null);
```

Comment:
I don't know, I don't have a personal preference here. Would you prefer `NoMetricsCollector`?